docs/admin/runai-setup/cluster-setup/cluster-install.md

```diff
@@ -17,7 +17,7 @@ Log in to Run:ai user interface at `<company-name>.run.ai`. Use credentials prov
 Using the cluster wizard:
 
 * Choose a name for your cluster.
-* Choose the Run:ai version for the cluster. 
+* Choose the Run:ai version for the cluster.
 * Choose a target Kubernetes distribution (see [table](cluster-prerequisites.md#kubernetes) for supported distributions).
 * (SaaS and remote self-hosted cluster only) Enter a URL for the Kubernetes cluster. The URL need only be accessible within the organization's network. For more informtaion see [here](cluster-prerequisites.md#cluster-url).
```

Comment:
@roir We generally do not use "here" but the name of the section. This is likely not your correction, but I fix these when I see them. 
```suggestion
* (SaaS and remote self-hosted cluster only) Enter a URL for the Kubernetes cluster. The URL need only be accessible within the organization's network. For more information, see [Cluster prerequisites](cluster-prerequisites.md#cluster-url).
```